git_machete/docs.py

```diff
@@ -298,19 +299,50 @@
     "go": """
         <b>Usage: git machete g[o] <direction></b>
         where <direction> is one of: `d[own]`, `f[irst]`, `l[ast]`, `n[ext]`, `p[rev]`, `r[oot]`, `u[p]`
-
         Checks out the branch specified by the given direction relative to the current branch:
-
         * `down`:    the direct children/downstream branch of the current branch.
         * `first`:   the first downstream of the root branch of the current branch (like `root` followed by `next`), or the root branch itself if the root has no downstream branches.
         * `last`:    the last branch in the definition file that has the same root as the current branch; can be the root branch itself if the root has no downstream branches.
         * `next`:    the direct successor of the current branch in the definition file.
         * `prev`:    the direct predecessor of the current branch in the definition file.
         * `root`:    the root of the tree where the current branch is located. Note: this will typically be something like `develop` or `master`, since all branches are usually meant to be ultimately merged to one of those.
         * `up`:      the direct parent/upstream branch of the current branch.
-
         Roughly equivalent to `git checkout $(git machete show <direction>)`.
     """,
+    "github": """
+        <b>Usage: git machete github <subcommand></b>
+        where <subcommand> is one of: `anno-prs`, `create-pr`, `retarget-pr`.
+
+        Creates, checks out and manages GitHub PRs while keeping them reflected in branch definition file.
+
+        To allow GitHub API access for private repositories (and also to correctly identify the current user, even in case of public repositories),
+        a GitHub API token with `repo` scope is required, see `https://github.com/settings/tokens`. This will be resolved from the first of:
+        1. `GITHUB_TOKEN` env var,
+        2. current auth token from the `gh` GitHub CLI,
+        3. current auth token from the `hub` GitHub CLI.
+
+        <b>`anno-prs`:</b>
+
+          Annotates the branches based on their corresponding GitHub PR numbers and authors.
+          Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
+          Equivalent to `git machete anno --sync-github-prs`.
```

Comment:
Is either `git machete github anno-prs` or `git machete anno --sync-github-prs` recommended? Maybe it would be possible/a good idea to deprecate one of the ways. It could happen for 4.0.0 version, but we could create a file/issue to store things that shall be removed in the future and in version 3.4.0 we could introduce deprecation warning.

Reply:
I'd say `git machete github anno-prs` is recommended... not sure about deprecating `git machete anno --sync-github-prs` tho 🤔

Reply:
lets keep both for now